lib/rubocop/token.rb

```diff
@@ -14,7 +14,8 @@ def self.from_parser_token(parser_token)
     def initialize(pos, type, text)
       @pos = pos
       @type = type
-      @text = text
+      # Parser token "text" may be a Fixnum
+      @text = text.is_a?(String) ? text : text.to_s
```

Comment:
Why not invoke `.to_s` on whatever `@text` is? No branching, pretty much the same result.